### PR TITLE
Add webUI tests for files sharing external

### PR DESCRIPTION
**Pull request by paurakhsharma**
## Description
The PR adds webUI acceptance tests for sharing activities related to external sharing.

## Related Issue
 #677 

## How Has This Been Tested?
:robot: 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

P.S: Some of the steps used here are yet to be merged in [PR: owncloud/core/pull/34294]

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -153,16 +181,26 @@ services:
...
 
-  owncloud:
+  server:
     image: owncloudci/php:${PHP_VERSION}
     pull: true
     environment:
-      - APACHE_WEBROOT=/var/www/owncloud/
-    command: ["/usr/local/bin/apachectl", "-e", "debug", "-D", "FOREGROUND"]
+      - APACHE_WEBROOT=/var/www/owncloud/server/
+    command: [ "/usr/local/bin/apachectl", "-e", "debug" , "-D" , "FOREGROUND" ]
```
Comment (dpakach) [inline .drone.yml]: spaces? why?

Comment (paurakhsharma) [reply .drone.yml]: It required some padding. jk. Will fix while squashing.

File: tests/acceptance/features/webUIActivitySharingExternal/sharingExternal.feature [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+  Scenario: Sharing a folder with a remote server should be listed in the activity list of a sharer
+    Given using server "REMOTE"
+    And parameter "auto_accept_trusted" of app "federatedfilesharing" has been set to "no"
+    And user "user1" has been created with default attributes
+    And user "user1" from server "LOCAL" has shared "simple-folder" with user "user1" from server "REMOTE"
+    When user "user1" from server "REMOTE" accepts the last pending share using the sharing API
+    And using server "LOCAL"
+    And user "user1" browses to the activity page
+    Then the activity number 1 should contain message "user1@… accepted remote share simple-folder" in the activity page
```
Comment (individual-it) [inline tests/acceptance/features/webUIActivitySharingExternal/sharingExternal.feature]: maybe we should use a different username on the remote server to be certain that the username comes from the remote server here

File: tests/acceptance/features/webUIActivitySharingExternal/sharingExternal.feature [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+  Scenario: Uploading a file to a public shared folder should be listed in the activity list
+    Given user "user1" has created a public link share with settings
+      | path        | simple-folder |
+      | permissions | create        |
+    And the public has uploaded file "test.txt" with content "This is a test"
+    When user "user1" browses to the activity page
+    Then the activity number 1 should contain message "created test.txt" in the activity page
+
+  Scenario: Downloading a file from a public shared folder using API should not be listed in the activity list
```
Comment (individual-it) [inline tests/acceptance/features/webUIActivitySharingExternal/sharingExternal.feature]: why should that activity not be listed?

**Comment by individual-it** [comment]
please also split the fed-sharing and public link sharing into two files

**Pull request merged**